src/convert.rs

```diff
@@ -0,0 +1,238 @@
+use num::Float;
+
+use {Alpha, Rgb, Luma, Xyz, Yxy, Lab, Lch, Hsv, Hsl, Color};
+
+///FromColor provides conversion between the colors. It requires from_xyz and derives conversion
```

Comment:
Would you mind separating the first sentence into its own paragraph? It will look nicer in the docs.